decentraland/unity-explorer
# fix: scene hover action tooltip filters

## What does this PR change?

Fixes #906 

## How to test the changes?

1. Follow issue steps
2. Check that all mouse hover scene interactions works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
